### PR TITLE
Fix and improvements

### PR DESCRIPTION
**Pull request by tyx**

### CODE REVIEW

File: Model/Violation.php [hunk 1]
```diff
@@ -304,13 +304,17 @@ public function getUpdatedAt()
...
+            $this->getMessage() === $violation->getMessage() &&
+            $this->getMessageParameters() === $violation->getMessageParameters()
+        ;
     }
 
-    /**
-     * @return array<string>
-     */
-    public function __sleep()
+    public function _sleep()
```
Comment (Spy-Seth) [inline Model/Violation.php]: Oups.

Comment (gmorel) [inline Model/Violation.php]: Typo or normal ? Only 1 underscore on sleep.

Comment (tyx) [inline Model/Violation.php]: bien vu l'aveugle

**Comment by Spy-Seth** [comment]
:+1:

**Pull request merged**